dotnet/minimal-api/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+
+services:
+  webapi:
+    ports:
+    - "8080:80"
+    build: .
+    env_file:
+    - .env
```

Comment:
```suggestion
    environment:
      OTEL_EXPORTER_OTLP_ENDPOINT: "${OTEL_EXPORTER_OTLP_ENDPOINT}"
      OTEL_EXPORTER_OTLP_HEADERS: "${OTEL_EXPORTER_OTLP_HEADERS}"
```

Reply:
Resolved in [321ae45](https://github.com/newrelic/newrelic-opentelemetry-examples/pull/99/commits/321ae452480435cbf14e03dcaea2ad8f59cdd8ce)